[2016-11-27 16:31:41] DEBUG [django.db.backends:90] (0.000) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
[2016-11-27 16:31:41] DEBUG [django.db.backends:90] (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
[2016-11-27 16:31:53] DEBUG [django.db.backends:90] (0.000) SELECT "django_session"."session_key", "django_session"."session_data", "django_session"."expire_date" FROM "django_session" WHERE ("django_session"."expire_date" > '2016-11-27 21:31:53.635542' AND "django_session"."session_key" = 'krsb3400kdtg4l0rixjy1o0j8epga26x'); args=('2016-11-27 21:31:53.635542', 'krsb3400kdtg4l0rixjy1o0j8epga26x')
[2016-11-27 16:31:53] DEBUG [django.db.backends:90] (0.000) SELECT "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by" FROM "auths_user" WHERE "auths_user"."id" = '01e659bc77a5436f9fc859d23a77e6aa'; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-11-27 16:31:53] DEBUG [django.template:929] Exception while resolving variable 'is_popup' in template 'admin/index.html'.
Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\context.py", line 75, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 891, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in "[{'None': None, 'False': False, 'True': True}, {'DEFAULT_MESSAGE_LEVELS': {'ERROR': 40, 'SUCCESS': 25, 'WARNING': 30, 'DEBUG': 10, 'INFO': 20}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x046C4B50>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x049D7690>, 'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x046C9660>>, 'user': <SimpleLazyObject: <User: admin>>, 'request': <WSGIRequest: GET '/admin/'>}, {}, {'available_apps': [{'app_label': 'auth', 'has_module_perms': True, 'models': [{'object_name': 'Group', 'admin_url': '/admin/auth/group/', 'add_url': '/admin/auth/group/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x049646D0>}, {'object_name': 'Permission', 'admin_url': '/admin/auth/permission/', 'add_url': '/admin/auth/permission/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x049646B0>}], 'app_url': '/admin/auth/', 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x03D70710>}, {'app_label': 'auths', 'has_module_perms': True, 'models': [{'object_name': 'HierarchyType', 'admin_url': '/admin/auths/hierarchytype/', 'add_url': '/admin/auths/hierarchytype/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04964470>}, {'object_name': 'Hierarchy', 'admin_url': '/admin/auths/hierarchy/', 'add_url': '/admin/auths/hierarchy/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x049649B0>}, {'object_name': 'Menu', 'admin_url': '/admin/auths/menu/', 'add_url': '/admin/auths/menu/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04964FB0>}, {'object_name': 'Person', 'admin_url': '/admin/auths/person/', 'add_url': '/admin/auths/person/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x049641F0>}, {'object_name': 'UserHierarchyGroup', 'admin_url': '/admin/auths/userhierarchygroup/', 'add_url': '/admin/auths/userhierarchygroup/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x049642B0>}, {'object_name': 'UserHierarchyPermission', 'admin_url': '/admin/auths/userhierarchypermission/', 'add_url': '/admin/auths/userhierarchypermission/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04964050>}, {'object_name': 'User', 'admin_url': '/admin/auths/user/', 'add_url': '/admin/auths/user/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04964BD0>}], 'app_url': '/admin/auths/', 'name': 'Auths'}, {'app_label': 'oauth2_provider', 'has_module_perms': True, 'models': [{'object_name': 'AccessToken', 'admin_url': '/admin/oauth2_provider/accesstoken/', 'add_url': '/admin/oauth2_provider/accesstoken/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04964490>}, {'object_name': 'Application', 'admin_url': '/admin/oauth2_provider/application/', 'add_url': '/admin/oauth2_provider/application/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04964290>}, {'object_name': 'Grant', 'admin_url': '/admin/oauth2_provider/grant/', 'add_url': '/admin/oauth2_provider/grant/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04948450>}, {'object_name': 'RefreshToken', 'admin_url': '/admin/oauth2_provider/refreshtoken/', 'add_url': '/admin/oauth2_provider/refreshtoken/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04964FD0>}], 'app_url': '/admin/oauth2_provider/', 'name': 'Django OAuth Toolkit'}, {'app_label': 'contenttypes', 'has_module_perms': True, 'models': [{'object_name': 'ContentType', 'admin_url': '/admin/contenttypes/contenttype/', 'add_url': '/admin/contenttypes/contenttype/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04948490>}], 'app_url': '/admin/contenttypes/', 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x03D70830>}, {'app_label': 'topico2', 'has_module_perms': True, 'models': [{'object_name': 'Consulta', 'admin_url': '/admin/topico2/consulta/', 'add_url': '/admin/topico2/consulta/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Consultas'}, {'object_name': 'Diagnostico', 'admin_url': '/admin/topico2/diagnostico/', 'add_url': '/admin/topico2/diagnostico/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Diagnosticos'}, {'object_name': 'EspecificacionReceta', 'admin_url': '/admin/topico2/especificacionreceta/', 'add_url': '/admin/topico2/especificacionreceta/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'EspecificacionRecetas'}, {'object_name': 'Historia', 'admin_url': '/admin/topico2/historia/', 'add_url': '/admin/topico2/historia/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Historias'}, {'object_name': 'Medicamento', 'admin_url': '/admin/topico2/medicamento/', 'add_url': '/admin/topico2/medicamento/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Medicamentos'}, {'object_name': 'Tratamiento', 'admin_url': '/admin/topico2/tratamiento/', 'add_url': '/admin/topico2/tratamiento/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Tratamientos'}], 'app_url': '/admin/topico2/', 'name': 'Topico2'}], 'site_url': '/', 'site_header': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x03D5CC70>, 'LANGUAGE_BIDI': False, 'app_list': [{'app_label': 'auth', 'has_module_perms': True, 'models': [{'object_name': 'Group', 'admin_url': '/admin/auth/group/', 'add_url': '/admin/auth/group/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953E10>}, {'object_name': 'Permission', 'admin_url': '/admin/auth/permission/', 'add_url': '/admin/auth/permission/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x049488D0>}], 'app_url': '/admin/auth/', 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x03D70710>}, {'app_label': 'auths', 'has_module_perms': True, 'models': [{'object_name': 'HierarchyType', 'admin_url': '/admin/auths/hierarchytype/', 'add_url': '/admin/auths/hierarchytype/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04948930>}, {'object_name': 'Hierarchy', 'admin_url': '/admin/auths/hierarchy/', 'add_url': '/admin/auths/hierarchy/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953FF0>}, {'object_name': 'Menu', 'admin_url': '/admin/auths/menu/', 'add_url': '/admin/auths/menu/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953E50>}, {'object_name': 'Person', 'admin_url': '/admin/auths/person/', 'add_url': '/admin/auths/person/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953E30>}, {'object_name': 'UserHierarchyGroup', 'admin_url': '/admin/auths/userhierarchygroup/', 'add_url': '/admin/auths/userhierarchygroup/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953DF0>}, {'object_name': 'UserHierarchyPermission', 'admin_url': '/admin/auths/userhierarchypermission/', 'add_url': '/admin/auths/userhierarchypermission/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x049483B0>}, {'object_name': 'User', 'admin_url': '/admin/auths/user/', 'add_url': '/admin/auths/user/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953FD0>}], 'app_url': '/admin/auths/', 'name': 'Auths'}, {'app_label': 'oauth2_provider', 'has_module_perms': True, 'models': [{'object_name': 'AccessToken', 'admin_url': '/admin/oauth2_provider/accesstoken/', 'add_url': '/admin/oauth2_provider/accesstoken/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953EB0>}, {'object_name': 'Application', 'admin_url': '/admin/oauth2_provider/application/', 'add_url': '/admin/oauth2_provider/application/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953ED0>}, {'object_name': 'Grant', 'admin_url': '/admin/oauth2_provider/grant/', 'add_url': '/admin/oauth2_provider/grant/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x046DBB10>}, {'object_name': 'RefreshToken', 'admin_url': '/admin/oauth2_provider/refreshtoken/', 'add_url': '/admin/oauth2_provider/refreshtoken/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953E90>}], 'app_url': '/admin/oauth2_provider/', 'name': 'Django OAuth Toolkit'}, {'app_label': 'contenttypes', 'has_module_perms': True, 'models': [{'object_name': 'ContentType', 'admin_url': '/admin/contenttypes/contenttype/', 'add_url': '/admin/contenttypes/contenttype/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x046DB810>}], 'app_url': '/admin/contenttypes/', 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x03D70830>}, {'app_label': 'topico2', 'has_module_perms': True, 'models': [{'object_name': 'Consulta', 'admin_url': '/admin/topico2/consulta/', 'add_url': '/admin/topico2/consulta/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Consultas'}, {'object_name': 'Diagnostico', 'admin_url': '/admin/topico2/diagnostico/', 'add_url': '/admin/topico2/diagnostico/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Diagnosticos'}, {'object_name': 'EspecificacionReceta', 'admin_url': '/admin/topico2/especificacionreceta/', 'add_url': '/admin/topico2/especificacionreceta/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'EspecificacionRecetas'}, {'object_name': 'Historia', 'admin_url': '/admin/topico2/historia/', 'add_url': '/admin/topico2/historia/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Historias'}, {'object_name': 'Medicamento', 'admin_url': '/admin/topico2/medicamento/', 'add_url': '/admin/topico2/medicamento/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Medicamentos'}, {'object_name': 'Tratamiento', 'admin_url': '/admin/topico2/tratamiento/', 'add_url': '/admin/topico2/tratamiento/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Tratamientos'}], 'app_url': '/admin/topico2/', 'name': 'Topico2'}], 'LANGUAGE_CODE': 'es', 'title': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x03D5CCD0>, 'site_title': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x03D5CC10>, 'has_permission': True}]"
[2016-11-27 16:31:53] DEBUG [django.template:929] Exception while resolving variable 'is_popup' in template 'admin/index.html'.
Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\context.py", line 75, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 891, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in "[{'None': None, 'False': False, 'True': True}, {'DEFAULT_MESSAGE_LEVELS': {'ERROR': 40, 'SUCCESS': 25, 'WARNING': 30, 'DEBUG': 10, 'INFO': 20}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x046C4B50>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x049D7690>, 'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x046C9660>>, 'user': <SimpleLazyObject: <User: admin>>, 'request': <WSGIRequest: GET '/admin/'>}, {}, {'available_apps': [{'app_label': 'auth', 'has_module_perms': True, 'models': [{'object_name': 'Group', 'admin_url': '/admin/auth/group/', 'add_url': '/admin/auth/group/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x049646D0>}, {'object_name': 'Permission', 'admin_url': '/admin/auth/permission/', 'add_url': '/admin/auth/permission/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x049646B0>}], 'app_url': '/admin/auth/', 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x03D70710>}, {'app_label': 'auths', 'has_module_perms': True, 'models': [{'object_name': 'HierarchyType', 'admin_url': '/admin/auths/hierarchytype/', 'add_url': '/admin/auths/hierarchytype/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04964470>}, {'object_name': 'Hierarchy', 'admin_url': '/admin/auths/hierarchy/', 'add_url': '/admin/auths/hierarchy/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x049649B0>}, {'object_name': 'Menu', 'admin_url': '/admin/auths/menu/', 'add_url': '/admin/auths/menu/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04964FB0>}, {'object_name': 'Person', 'admin_url': '/admin/auths/person/', 'add_url': '/admin/auths/person/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x049641F0>}, {'object_name': 'UserHierarchyGroup', 'admin_url': '/admin/auths/userhierarchygroup/', 'add_url': '/admin/auths/userhierarchygroup/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x049642B0>}, {'object_name': 'UserHierarchyPermission', 'admin_url': '/admin/auths/userhierarchypermission/', 'add_url': '/admin/auths/userhierarchypermission/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04964050>}, {'object_name': 'User', 'admin_url': '/admin/auths/user/', 'add_url': '/admin/auths/user/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04964BD0>}], 'app_url': '/admin/auths/', 'name': 'Auths'}, {'app_label': 'oauth2_provider', 'has_module_perms': True, 'models': [{'object_name': 'AccessToken', 'admin_url': '/admin/oauth2_provider/accesstoken/', 'add_url': '/admin/oauth2_provider/accesstoken/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04964490>}, {'object_name': 'Application', 'admin_url': '/admin/oauth2_provider/application/', 'add_url': '/admin/oauth2_provider/application/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04964290>}, {'object_name': 'Grant', 'admin_url': '/admin/oauth2_provider/grant/', 'add_url': '/admin/oauth2_provider/grant/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04948450>}, {'object_name': 'RefreshToken', 'admin_url': '/admin/oauth2_provider/refreshtoken/', 'add_url': '/admin/oauth2_provider/refreshtoken/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04964FD0>}], 'app_url': '/admin/oauth2_provider/', 'name': 'Django OAuth Toolkit'}, {'app_label': 'contenttypes', 'has_module_perms': True, 'models': [{'object_name': 'ContentType', 'admin_url': '/admin/contenttypes/contenttype/', 'add_url': '/admin/contenttypes/contenttype/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04948490>}], 'app_url': '/admin/contenttypes/', 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x03D70830>}, {'app_label': 'topico2', 'has_module_perms': True, 'models': [{'object_name': 'Consulta', 'admin_url': '/admin/topico2/consulta/', 'add_url': '/admin/topico2/consulta/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Consultas'}, {'object_name': 'Diagnostico', 'admin_url': '/admin/topico2/diagnostico/', 'add_url': '/admin/topico2/diagnostico/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Diagnosticos'}, {'object_name': 'EspecificacionReceta', 'admin_url': '/admin/topico2/especificacionreceta/', 'add_url': '/admin/topico2/especificacionreceta/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'EspecificacionRecetas'}, {'object_name': 'Historia', 'admin_url': '/admin/topico2/historia/', 'add_url': '/admin/topico2/historia/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Historias'}, {'object_name': 'Medicamento', 'admin_url': '/admin/topico2/medicamento/', 'add_url': '/admin/topico2/medicamento/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Medicamentos'}, {'object_name': 'Tratamiento', 'admin_url': '/admin/topico2/tratamiento/', 'add_url': '/admin/topico2/tratamiento/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Tratamientos'}], 'app_url': '/admin/topico2/', 'name': 'Topico2'}], 'site_url': '/', 'site_header': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x03D5CC70>, 'LANGUAGE_BIDI': False, 'app_list': [{'app_label': 'auth', 'has_module_perms': True, 'models': [{'object_name': 'Group', 'admin_url': '/admin/auth/group/', 'add_url': '/admin/auth/group/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953E10>}, {'object_name': 'Permission', 'admin_url': '/admin/auth/permission/', 'add_url': '/admin/auth/permission/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x049488D0>}], 'app_url': '/admin/auth/', 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x03D70710>}, {'app_label': 'auths', 'has_module_perms': True, 'models': [{'object_name': 'HierarchyType', 'admin_url': '/admin/auths/hierarchytype/', 'add_url': '/admin/auths/hierarchytype/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04948930>}, {'object_name': 'Hierarchy', 'admin_url': '/admin/auths/hierarchy/', 'add_url': '/admin/auths/hierarchy/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953FF0>}, {'object_name': 'Menu', 'admin_url': '/admin/auths/menu/', 'add_url': '/admin/auths/menu/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953E50>}, {'object_name': 'Person', 'admin_url': '/admin/auths/person/', 'add_url': '/admin/auths/person/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953E30>}, {'object_name': 'UserHierarchyGroup', 'admin_url': '/admin/auths/userhierarchygroup/', 'add_url': '/admin/auths/userhierarchygroup/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953DF0>}, {'object_name': 'UserHierarchyPermission', 'admin_url': '/admin/auths/userhierarchypermission/', 'add_url': '/admin/auths/userhierarchypermission/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x049483B0>}, {'object_name': 'User', 'admin_url': '/admin/auths/user/', 'add_url': '/admin/auths/user/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953FD0>}], 'app_url': '/admin/auths/', 'name': 'Auths'}, {'app_label': 'oauth2_provider', 'has_module_perms': True, 'models': [{'object_name': 'AccessToken', 'admin_url': '/admin/oauth2_provider/accesstoken/', 'add_url': '/admin/oauth2_provider/accesstoken/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953EB0>}, {'object_name': 'Application', 'admin_url': '/admin/oauth2_provider/application/', 'add_url': '/admin/oauth2_provider/application/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953ED0>}, {'object_name': 'Grant', 'admin_url': '/admin/oauth2_provider/grant/', 'add_url': '/admin/oauth2_provider/grant/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x046DBB10>}, {'object_name': 'RefreshToken', 'admin_url': '/admin/oauth2_provider/refreshtoken/', 'add_url': '/admin/oauth2_provider/refreshtoken/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04953E90>}], 'app_url': '/admin/oauth2_provider/', 'name': 'Django OAuth Toolkit'}, {'app_label': 'contenttypes', 'has_module_perms': True, 'models': [{'object_name': 'ContentType', 'admin_url': '/admin/contenttypes/contenttype/', 'add_url': '/admin/contenttypes/contenttype/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x046DB810>}], 'app_url': '/admin/contenttypes/', 'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x03D70830>}, {'app_label': 'topico2', 'has_module_perms': True, 'models': [{'object_name': 'Consulta', 'admin_url': '/admin/topico2/consulta/', 'add_url': '/admin/topico2/consulta/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Consultas'}, {'object_name': 'Diagnostico', 'admin_url': '/admin/topico2/diagnostico/', 'add_url': '/admin/topico2/diagnostico/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Diagnosticos'}, {'object_name': 'EspecificacionReceta', 'admin_url': '/admin/topico2/especificacionreceta/', 'add_url': '/admin/topico2/especificacionreceta/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'EspecificacionRecetas'}, {'object_name': 'Historia', 'admin_url': '/admin/topico2/historia/', 'add_url': '/admin/topico2/historia/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Historias'}, {'object_name': 'Medicamento', 'admin_url': '/admin/topico2/medicamento/', 'add_url': '/admin/topico2/medicamento/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Medicamentos'}, {'object_name': 'Tratamiento', 'admin_url': '/admin/topico2/tratamiento/', 'add_url': '/admin/topico2/tratamiento/add/', 'perms': {'change': True, 'add': True, 'delete': True}, 'name': 'Tratamientos'}], 'app_url': '/admin/topico2/', 'name': 'Topico2'}], 'LANGUAGE_CODE': 'es', 'title': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x03D5CCD0>, 'site_title': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x03D5CC10>, 'has_permission': True}]"
[2016-11-27 16:31:53] DEBUG [django.db.backends:90] (0.003) SELECT "django_admin_log"."id", "django_admin_log"."action_time", "django_admin_log"."user_id", "django_admin_log"."content_type_id", "django_admin_log"."object_id", "django_admin_log"."object_repr", "django_admin_log"."action_flag", "django_admin_log"."change_message", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "django_content_type"."id", "django_content_type"."app_label", "django_content_type"."model" FROM "django_admin_log" INNER JOIN "auths_user" ON ("django_admin_log"."user_id" = "auths_user"."id") LEFT OUTER JOIN "django_content_type" ON ("django_admin_log"."content_type_id" = "django_content_type"."id") WHERE "django_admin_log"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa' ORDER BY "django_admin_log"."action_time" DESC LIMIT 10; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-11-27 16:31:53] INFO [django.server:131] "GET /admin/ HTTP/1.1" 200 14999
[2016-11-27 16:31:53] INFO [django.server:131] "GET /static/admin/css/base.css HTTP/1.1" 304 0
[2016-11-27 16:31:54] INFO [django.server:131] "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
[2016-11-27 16:31:54] INFO [django.server:131] "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
[2016-11-27 16:31:54] INFO [django.server:131] "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
[2016-11-27 16:31:54] INFO [django.server:131] "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
[2016-11-27 16:31:54] INFO [django.server:131] "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
[2016-11-27 16:31:54] INFO [django.server:131] "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
[2016-11-27 16:31:54] INFO [django.server:131] "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
[2016-11-27 16:31:54] DEBUG [django.template:929] Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\exception.py", line 39, in inner
    response = get_response(request)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\urls\resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'django.contrib.admindocs.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\django\\contrib\\admindocs\\urls.py'> (None:None) ^admin/doc/>], [<RegexURLResolver <module 'rest_framework.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\rest_framework\\urls.py'> (rest_framework:rest_framework) ^api-auth/>], [<RegexURLResolver <module 'oauth2_provider.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\oauth2_provider\\urls.py'> (None:oauth2_provider) ^o/>], [<RegexURLResolver <module 'medicfastv2_service_apps.auths_api.urls' from 'D:\\lugar\\medicfastv2\\medicfastv2_service\\medicfastv2_service_apps\\auths_api\\urls.py'> (None:None) ^api/auths/>], [<RegexURLResolver <module 'medicfastv2_service_apps.topico2_api.urls' from 'D:\\lugar\\medicfastv2\\medicfastv2_service\\medicfastv2_service_apps\\topico2_api\\urls.py'> (None:None) ^api/topico2/>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in '<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>'
[2016-11-27 16:31:54] DEBUG [django.template:929] Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\exception.py", line 39, in inner
    response = get_response(request)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\urls\resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'django.contrib.admindocs.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\django\\contrib\\admindocs\\urls.py'> (None:None) ^admin/doc/>], [<RegexURLResolver <module 'rest_framework.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\rest_framework\\urls.py'> (rest_framework:rest_framework) ^api-auth/>], [<RegexURLResolver <module 'oauth2_provider.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\oauth2_provider\\urls.py'> (None:oauth2_provider) ^o/>], [<RegexURLResolver <module 'medicfastv2_service_apps.auths_api.urls' from 'D:\\lugar\\medicfastv2\\medicfastv2_service\\medicfastv2_service_apps\\auths_api\\urls.py'> (None:None) ^api/auths/>], [<RegexURLResolver <module 'medicfastv2_service_apps.topico2_api.urls' from 'D:\\lugar\\medicfastv2\\medicfastv2_service\\medicfastv2_service_apps\\topico2_api\\urls.py'> (None:None) ^api/topico2/>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'django.contrib.admindocs.urls' from 'D:\\\\lugar\\\\medicfastv2\\\\ve_medicfast\\\\lib\\\\site-packages\\\\django\\\\contrib\\\\admindocs\\\\urls.py'> (None:None) ^admin/doc/>"
[2016-11-27 16:31:54] DEBUG [django.template:929] Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\exception.py", line 39, in inner
    response = get_response(request)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\urls\resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'django.contrib.admindocs.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\django\\contrib\\admindocs\\urls.py'> (None:None) ^admin/doc/>], [<RegexURLResolver <module 'rest_framework.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\rest_framework\\urls.py'> (rest_framework:rest_framework) ^api-auth/>], [<RegexURLResolver <module 'oauth2_provider.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\oauth2_provider\\urls.py'> (None:oauth2_provider) ^o/>], [<RegexURLResolver <module 'medicfastv2_service_apps.auths_api.urls' from 'D:\\lugar\\medicfastv2\\medicfastv2_service\\medicfastv2_service_apps\\auths_api\\urls.py'> (None:None) ^api/auths/>], [<RegexURLResolver <module 'medicfastv2_service_apps.topico2_api.urls' from 'D:\\lugar\\medicfastv2\\medicfastv2_service\\medicfastv2_service_apps\\topico2_api\\urls.py'> (None:None) ^api/topico2/>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'rest_framework.urls' from 'D:\\\\lugar\\\\medicfastv2\\\\ve_medicfast\\\\lib\\\\site-packages\\\\rest_framework\\\\urls.py'> (rest_framework:rest_framework) ^api-auth/>"
[2016-11-27 16:31:54] DEBUG [django.template:929] Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\exception.py", line 39, in inner
    response = get_response(request)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\urls\resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'django.contrib.admindocs.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\django\\contrib\\admindocs\\urls.py'> (None:None) ^admin/doc/>], [<RegexURLResolver <module 'rest_framework.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\rest_framework\\urls.py'> (rest_framework:rest_framework) ^api-auth/>], [<RegexURLResolver <module 'oauth2_provider.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\oauth2_provider\\urls.py'> (None:oauth2_provider) ^o/>], [<RegexURLResolver <module 'medicfastv2_service_apps.auths_api.urls' from 'D:\\lugar\\medicfastv2\\medicfastv2_service\\medicfastv2_service_apps\\auths_api\\urls.py'> (None:None) ^api/auths/>], [<RegexURLResolver <module 'medicfastv2_service_apps.topico2_api.urls' from 'D:\\lugar\\medicfastv2\\medicfastv2_service\\medicfastv2_service_apps\\topico2_api\\urls.py'> (None:None) ^api/topico2/>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'oauth2_provider.urls' from 'D:\\\\lugar\\\\medicfastv2\\\\ve_medicfast\\\\lib\\\\site-packages\\\\oauth2_provider\\\\urls.py'> (None:oauth2_provider) ^o/>"
[2016-11-27 16:31:54] DEBUG [django.template:929] Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\exception.py", line 39, in inner
    response = get_response(request)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\urls\resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'django.contrib.admindocs.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\django\\contrib\\admindocs\\urls.py'> (None:None) ^admin/doc/>], [<RegexURLResolver <module 'rest_framework.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\rest_framework\\urls.py'> (rest_framework:rest_framework) ^api-auth/>], [<RegexURLResolver <module 'oauth2_provider.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\oauth2_provider\\urls.py'> (None:oauth2_provider) ^o/>], [<RegexURLResolver <module 'medicfastv2_service_apps.auths_api.urls' from 'D:\\lugar\\medicfastv2\\medicfastv2_service\\medicfastv2_service_apps\\auths_api\\urls.py'> (None:None) ^api/auths/>], [<RegexURLResolver <module 'medicfastv2_service_apps.topico2_api.urls' from 'D:\\lugar\\medicfastv2\\medicfastv2_service\\medicfastv2_service_apps\\topico2_api\\urls.py'> (None:None) ^api/topico2/>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'medicfastv2_service_apps.auths_api.urls' from 'D:\\\\lugar\\\\medicfastv2\\\\medicfastv2_service\\\\medicfastv2_service_apps\\\\auths_api\\\\urls.py'> (None:None) ^api/auths/>"
[2016-11-27 16:31:54] DEBUG [django.template:929] Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\exception.py", line 39, in inner
    response = get_response(request)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\urls\resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'django.contrib.admindocs.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\django\\contrib\\admindocs\\urls.py'> (None:None) ^admin/doc/>], [<RegexURLResolver <module 'rest_framework.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\rest_framework\\urls.py'> (rest_framework:rest_framework) ^api-auth/>], [<RegexURLResolver <module 'oauth2_provider.urls' from 'D:\\lugar\\medicfastv2\\ve_medicfast\\lib\\site-packages\\oauth2_provider\\urls.py'> (None:oauth2_provider) ^o/>], [<RegexURLResolver <module 'medicfastv2_service_apps.auths_api.urls' from 'D:\\lugar\\medicfastv2\\medicfastv2_service\\medicfastv2_service_apps\\auths_api\\urls.py'> (None:None) ^api/auths/>], [<RegexURLResolver <module 'medicfastv2_service_apps.topico2_api.urls' from 'D:\\lugar\\medicfastv2\\medicfastv2_service\\medicfastv2_service_apps\\topico2_api\\urls.py'> (None:None) ^api/topico2/>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\lugar\medicfastv2\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'medicfastv2_service_apps.topico2_api.urls' from 'D:\\\\lugar\\\\medicfastv2\\\\medicfastv2_service\\\\medicfastv2_service_apps\\\\topico2_api\\\\urls.py'> (None:None) ^api/topico2/>"
[2016-11-27 16:31:54] WARNING [django.request:152] Not Found: /favicon.ico
[2016-11-27 16:31:54] WARNING [django.server:131] "GET /favicon.ico HTTP/1.1" 404 2494
[2016-11-27 16:32:02] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-11-27 16:32:02] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-11-27 16:32:21] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_application" WHERE "oauth2_provider_application"."client_id" = 'XHJWk31Fzj0uLmx49TZ8zJ1Dti31HAUTKrPNuf65'; args=('XHJWk31Fzj0uLmx49TZ8zJ1Dti31HAUTKrPNuf65',)
[2016-11-27 16:32:21] DEBUG [django.db.backends:90] (0.000) SELECT "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by" FROM "auths_user" WHERE "auths_user"."username" = 'admin'; args=('admin',)
[2016-11-27 16:32:21] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-11-27 16:32:21] DEBUG [django.db.backends:90] (0.001) INSERT INTO "oauth2_provider_accesstoken" ("user_id", "token", "application_id", "expires", "scope") VALUES ('01e659bc77a5436f9fc859d23a77e6aa', 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2', 1, '2016-11-28 07:32:21.208802', 'read write'); args=['01e659bc77a5436f9fc859d23a77e6aa', 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2', 1, '2016-11-28 07:32:21.208802', 'read write']
[2016-11-27 16:32:21] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-11-27 16:32:21] DEBUG [django.db.backends:90] (0.001) INSERT INTO "oauth2_provider_refreshtoken" ("user_id", "token", "application_id", "access_token_id") VALUES ('01e659bc77a5436f9fc859d23a77e6aa', 'OtbSyYmJ5h1en3L4Zt2QOokudPuQ0Y', 1, 18); args=['01e659bc77a5436f9fc859d23a77e6aa', 'OtbSyYmJ5h1en3L4Zt2QOokudPuQ0Y', 1, 18]
[2016-11-27 16:32:21] INFO [django.server:131] "POST /o/token/ HTTP/1.1" 200 169
[2016-11-27 16:32:22] INFO [django.server:131] "OPTIONS /api/auths/localuserinfo/ HTTP/1.1" 200 0
[2016-11-27 16:32:22] INFO [django.server:131] "OPTIONS /api/auths/usermenu/ HTTP/1.1" 200 0
[2016-11-27 16:32:22] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:32:22] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:32:22] DEBUG [django.db.backends:90] (0.005) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:32:22] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:32:22] DEBUG [django.db.backends:90] (0.006) SELECT "auth_group"."id", "auth_group"."name" FROM "auth_group" INNER JOIN "auths_user_groups" ON ("auth_group"."id" = "auths_user_groups"."group_id") WHERE "auths_user_groups"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa'; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-11-27 16:32:22] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."id" FROM "auths_menu" WHERE ("auths_menu"."id" IS NOT NULL AND "auths_menu"."module" = 'BACKEND' AND "auths_menu"."is_active" = 1) ORDER BY "auths_menu"."pos" ASC; args=('BACKEND', True)
[2016-11-27 16:32:22] DEBUG [django.db.backends:90] (0.002) SELECT "auth_permission"."id", "auth_permission"."name", "auth_permission"."content_type_id", "auth_permission"."codename" FROM "auth_permission" INNER JOIN "auths_user_user_permissions" ON ("auth_permission"."id" = "auths_user_user_permissions"."permission_id") INNER JOIN "django_content_type" ON ("auth_permission"."content_type_id" = "django_content_type"."id") WHERE "auths_user_user_permissions"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa' ORDER BY "django_content_type"."app_label" ASC, "django_content_type"."model" ASC, "auth_permission"."codename" ASC; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-11-27 16:32:22] DEBUG [django.db.backends:90] (0.001) SELECT (1) AS "a" FROM "django_session" WHERE "django_session"."session_key" = 'nor3qc5cmxrd6iat1l5pyj6j3uj29ad2' LIMIT 1; args=('nor3qc5cmxrd6iat1l5pyj6j3uj29ad2',)
[2016-11-27 16:32:22] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-11-27 16:32:22] INFO [django.server:131] "GET /api/auths/localuserinfo/ HTTP/1.1" 200 405
[2016-11-27 16:32:22] DEBUG [django.db.backends:90] (0.001) INSERT INTO "django_session" ("session_key", "session_data", "expire_date") SELECT 'nor3qc5cmxrd6iat1l5pyj6j3uj29ad2', 'ZWMyMmE2MTA1MzM3NDcxZTRmYzk5YzliZTNiYjg2ZTZkMjFlMTc0NTp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-11-27 21:32:52.148081'; args=('nor3qc5cmxrd6iat1l5pyj6j3uj29ad2', 'ZWMyMmE2MTA1MzM3NDcxZTRmYzk5YzliZTNiYjg2ZTZkMjFlMTc0NTp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-11-27 21:32:52.148081')
[2016-11-27 16:32:22] INFO [django.server:131] "GET /api/auths/usermenu/ HTTP/1.1" 200 11
[2016-11-27 16:32:27] INFO [django.server:131] "OPTIONS /api/topico2/diagnosticos/ HTTP/1.1" 200 0
[2016-11-27 16:32:28] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:32:28] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:32:28] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_diagnostico"."id", "topico2_diagnostico"."codigo", "topico2_diagnostico"."nombre" FROM "topico2_diagnostico"; args=()
[2016-11-27 16:32:28] INFO [django.server:131] "GET /api/topico2/diagnosticos/ HTTP/1.1" 200 254
[2016-11-27 16:33:05] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:33:05] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:33:05] DEBUG [django.db.backends:90] (0.009) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:33:05] DEBUG [django.db.backends:90] (0.001) SELECT "auth_group"."id", "auth_group"."name" FROM "auth_group" INNER JOIN "auths_user_groups" ON ("auth_group"."id" = "auths_user_groups"."group_id") WHERE "auths_user_groups"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa'; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-11-27 16:33:05] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:33:05] DEBUG [django.db.backends:90] (0.000) SELECT "auth_permission"."id", "auth_permission"."name", "auth_permission"."content_type_id", "auth_permission"."codename" FROM "auth_permission" INNER JOIN "auths_user_user_permissions" ON ("auth_permission"."id" = "auths_user_user_permissions"."permission_id") INNER JOIN "django_content_type" ON ("auth_permission"."content_type_id" = "django_content_type"."id") WHERE "auths_user_user_permissions"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa' ORDER BY "django_content_type"."app_label" ASC, "django_content_type"."model" ASC, "auth_permission"."codename" ASC; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-11-27 16:33:05] DEBUG [django.db.backends:90] (0.000) SELECT "auths_menu"."id" FROM "auths_menu" WHERE ("auths_menu"."id" IS NOT NULL AND "auths_menu"."module" = 'BACKEND' AND "auths_menu"."is_active" = 1) ORDER BY "auths_menu"."pos" ASC; args=('BACKEND', True)
[2016-11-27 16:33:05] INFO [django.server:131] "GET /api/auths/localuserinfo/ HTTP/1.1" 200 405
[2016-11-27 16:33:05] DEBUG [django.db.backends:90] (0.000) SELECT (1) AS "a" FROM "django_session" WHERE "django_session"."session_key" = 'bmex8uzu0v4c9wn4yyqcd2e8rdu8rhqd' LIMIT 1; args=('bmex8uzu0v4c9wn4yyqcd2e8rdu8rhqd',)
[2016-11-27 16:33:05] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-11-27 16:33:05] DEBUG [django.db.backends:90] (0.001) INSERT INTO "django_session" ("session_key", "session_data", "expire_date") SELECT 'bmex8uzu0v4c9wn4yyqcd2e8rdu8rhqd', 'ZWMyMmE2MTA1MzM3NDcxZTRmYzk5YzliZTNiYjg2ZTZkMjFlMTc0NTp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-11-27 21:33:35.680033'; args=('bmex8uzu0v4c9wn4yyqcd2e8rdu8rhqd', 'ZWMyMmE2MTA1MzM3NDcxZTRmYzk5YzliZTNiYjg2ZTZkMjFlMTc0NTp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-11-27 21:33:35.680033')
[2016-11-27 16:33:05] INFO [django.server:131] "GET /api/auths/usermenu/ HTTP/1.1" 200 11
[2016-11-27 16:33:05] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:33:05] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:33:05] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_diagnostico"."id", "topico2_diagnostico"."codigo", "topico2_diagnostico"."nombre" FROM "topico2_diagnostico"; args=()
[2016-11-27 16:33:05] INFO [django.server:131] "GET /api/topico2/diagnosticos/ HTTP/1.1" 200 254
[2016-11-27 16:41:40] DEBUG [django.db.backends:90] (0.003) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:41:40] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:41:40] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_diagnostico"."id", "topico2_diagnostico"."codigo", "topico2_diagnostico"."nombre" FROM "topico2_diagnostico"; args=()
[2016-11-27 16:41:40] INFO [django.server:131] "GET /api/topico2/diagnosticos/ HTTP/1.1" 200 254
[2016-11-27 16:41:42] INFO [django.server:131] "OPTIONS /api/topico2/historias/ HTTP/1.1" 200 0
[2016-11-27 16:41:42] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:41:42] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:41:42] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_historia"."id", "topico2_historia"."numero", "topico2_historia"."atencion", "topico2_historia"."estado", "topico2_historia"."created_at", "topico2_historia"."updated_at" FROM "topico2_historia"; args=()
[2016-11-27 16:41:42] INFO [django.server:131] "GET /api/topico2/historias/ HTTP/1.1" 200 1290
[2016-11-27 16:43:40] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:43:40] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:43:40] DEBUG [django.db.backends:90] (0.010) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:43:40] DEBUG [django.db.backends:90] (0.000) SELECT "auth_group"."id", "auth_group"."name" FROM "auth_group" INNER JOIN "auths_user_groups" ON ("auth_group"."id" = "auths_user_groups"."group_id") WHERE "auths_user_groups"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa'; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-11-27 16:43:40] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:43:40] DEBUG [django.db.backends:90] (0.001) SELECT "auth_permission"."id", "auth_permission"."name", "auth_permission"."content_type_id", "auth_permission"."codename" FROM "auth_permission" INNER JOIN "auths_user_user_permissions" ON ("auth_permission"."id" = "auths_user_user_permissions"."permission_id") INNER JOIN "django_content_type" ON ("auth_permission"."content_type_id" = "django_content_type"."id") WHERE "auths_user_user_permissions"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa' ORDER BY "django_content_type"."app_label" ASC, "django_content_type"."model" ASC, "auth_permission"."codename" ASC; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-11-27 16:43:40] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."id" FROM "auths_menu" WHERE ("auths_menu"."id" IS NOT NULL AND "auths_menu"."module" = 'BACKEND' AND "auths_menu"."is_active" = 1) ORDER BY "auths_menu"."pos" ASC; args=('BACKEND', True)
[2016-11-27 16:43:40] INFO [django.server:131] "GET /api/auths/localuserinfo/ HTTP/1.1" 200 405
[2016-11-27 16:43:40] DEBUG [django.db.backends:90] (0.000) SELECT (1) AS "a" FROM "django_session" WHERE "django_session"."session_key" = 'zy4oeo4zyh0f4jupn6eorruzdummurh5' LIMIT 1; args=('zy4oeo4zyh0f4jupn6eorruzdummurh5',)
[2016-11-27 16:43:40] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-11-27 16:43:40] DEBUG [django.db.backends:90] (0.001) INSERT INTO "django_session" ("session_key", "session_data", "expire_date") SELECT 'zy4oeo4zyh0f4jupn6eorruzdummurh5', 'ZWMyMmE2MTA1MzM3NDcxZTRmYzk5YzliZTNiYjg2ZTZkMjFlMTc0NTp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-11-27 21:44:10.252832'; args=('zy4oeo4zyh0f4jupn6eorruzdummurh5', 'ZWMyMmE2MTA1MzM3NDcxZTRmYzk5YzliZTNiYjg2ZTZkMjFlMTc0NTp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-11-27 21:44:10.252832')
[2016-11-27 16:43:40] INFO [django.server:131] "GET /api/auths/usermenu/ HTTP/1.1" 200 11
[2016-11-27 16:43:48] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:43:48] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:43:48] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-11-27 16:43:48] DEBUG [django.db.backends:90] (0.000) INSERT INTO "topico2_historia" ("numero", "atencion", "estado", "created_at", "updated_at") VALUES ('21345', 1, 1, '2016-11-27 21:43:48.664297', '2016-11-27 21:43:48.665298'); args=['21345', True, True, '2016-11-27 21:43:48.664297', '2016-11-27 21:43:48.665298']
[2016-11-27 16:43:48] INFO [django.server:131] "POST /api/topico2/historias/ HTTP/1.1" 201 142
[2016-11-27 16:43:49] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:43:49] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 16:43:49] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_historia"."id", "topico2_historia"."numero", "topico2_historia"."atencion", "topico2_historia"."estado", "topico2_historia"."created_at", "topico2_historia"."updated_at" FROM "topico2_historia"; args=()
[2016-11-27 16:43:49] INFO [django.server:131] "GET /api/topico2/historias/ HTTP/1.1" 200 1433
[2016-11-27 17:18:44] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:44] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:44] DEBUG [django.db.backends:90] (0.007) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:44] DEBUG [django.db.backends:90] (0.003) SELECT "auth_group"."id", "auth_group"."name" FROM "auth_group" INNER JOIN "auths_user_groups" ON ("auth_group"."id" = "auths_user_groups"."group_id") WHERE "auths_user_groups"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa'; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-11-27 17:18:44] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:44] DEBUG [django.db.backends:90] (0.001) SELECT "auth_permission"."id", "auth_permission"."name", "auth_permission"."content_type_id", "auth_permission"."codename" FROM "auth_permission" INNER JOIN "auths_user_user_permissions" ON ("auth_permission"."id" = "auths_user_user_permissions"."permission_id") INNER JOIN "django_content_type" ON ("auth_permission"."content_type_id" = "django_content_type"."id") WHERE "auths_user_user_permissions"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa' ORDER BY "django_content_type"."app_label" ASC, "django_content_type"."model" ASC, "auth_permission"."codename" ASC; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-11-27 17:18:44] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."id" FROM "auths_menu" WHERE ("auths_menu"."id" IS NOT NULL AND "auths_menu"."module" = 'BACKEND' AND "auths_menu"."is_active" = 1) ORDER BY "auths_menu"."pos" ASC; args=('BACKEND', True)
[2016-11-27 17:18:44] INFO [django.server:131] "GET /api/auths/localuserinfo/ HTTP/1.1" 200 405
[2016-11-27 17:18:44] DEBUG [django.db.backends:90] (0.000) SELECT (1) AS "a" FROM "django_session" WHERE "django_session"."session_key" = '37gkmj3w0kf90ywtkdkrdaptbgaltmh7' LIMIT 1; args=('37gkmj3w0kf90ywtkdkrdaptbgaltmh7',)
[2016-11-27 17:18:44] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-11-27 17:18:44] DEBUG [django.db.backends:90] (0.001) INSERT INTO "django_session" ("session_key", "session_data", "expire_date") SELECT '37gkmj3w0kf90ywtkdkrdaptbgaltmh7', 'ZWMyMmE2MTA1MzM3NDcxZTRmYzk5YzliZTNiYjg2ZTZkMjFlMTc0NTp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-11-27 22:19:14.951784'; args=('37gkmj3w0kf90ywtkdkrdaptbgaltmh7', 'ZWMyMmE2MTA1MzM3NDcxZTRmYzk5YzliZTNiYjg2ZTZkMjFlMTc0NTp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-11-27 22:19:14.951784')
[2016-11-27 17:18:45] INFO [django.server:131] "GET /api/auths/usermenu/ HTTP/1.1" 200 11
[2016-11-27 17:18:47] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:47] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:47] DEBUG [django.db.backends:90] (0.001) SELECT "topico2_historia"."id", "topico2_historia"."numero", "topico2_historia"."atencion", "topico2_historia"."estado", "topico2_historia"."created_at", "topico2_historia"."updated_at" FROM "topico2_historia"; args=()
[2016-11-27 17:18:47] INFO [django.server:131] "GET /api/topico2/historias/ HTTP/1.1" 200 1433
[2016-11-27 17:18:50] INFO [django.server:131] "OPTIONS /api/topico2/consultas/ HTTP/1.1" 200 0
[2016-11-27 17:18:50] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:50] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:50] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_consulta"."id", "topico2_consulta"."historia_id", "topico2_consulta"."enfermedadactual", "topico2_consulta"."examenfisico", "topico2_consulta"."fecha", "topico2_consulta"."suministrar", "topico2_consulta"."estado" FROM "topico2_consulta"; args=()
[2016-11-27 17:18:50] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_diagnostico"."id", "topico2_diagnostico"."codigo", "topico2_diagnostico"."nombre" FROM "topico2_diagnostico" INNER JOIN "topico2_consulta_diagnostico" ON ("topico2_diagnostico"."id" = "topico2_consulta_diagnostico"."diagnostico_id") WHERE "topico2_consulta_diagnostico"."consulta_id" = 4; args=(4,)
[2016-11-27 17:18:50] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_diagnostico"."id", "topico2_diagnostico"."codigo", "topico2_diagnostico"."nombre" FROM "topico2_diagnostico" INNER JOIN "topico2_consulta_diagnostico" ON ("topico2_diagnostico"."id" = "topico2_consulta_diagnostico"."diagnostico_id") WHERE "topico2_consulta_diagnostico"."consulta_id" = 5; args=(5,)
[2016-11-27 17:18:50] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_diagnostico"."id", "topico2_diagnostico"."codigo", "topico2_diagnostico"."nombre" FROM "topico2_diagnostico" INNER JOIN "topico2_consulta_diagnostico" ON ("topico2_diagnostico"."id" = "topico2_consulta_diagnostico"."diagnostico_id") WHERE "topico2_consulta_diagnostico"."consulta_id" = 6; args=(6,)
[2016-11-27 17:18:50] INFO [django.server:131] "GET /api/topico2/consultas/ HTTP/1.1" 200 540
[2016-11-27 17:18:51] INFO [django.server:131] "OPTIONS /api/topico2/tratamientos/ HTTP/1.1" 200 0
[2016-11-27 17:18:51] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:51] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:51] DEBUG [django.db.backends:90] (0.001) SELECT "topico2_tratamiento"."id", "topico2_tratamiento"."consulta_id", "topico2_tratamiento"."diagnostico", "topico2_tratamiento"."periododescanso", "topico2_tratamiento"."fecha", "topico2_tratamiento"."citafecha", "topico2_tratamiento"."aministracion" FROM "topico2_tratamiento"; args=()
[2016-11-27 17:18:51] INFO [django.server:131] "GET /api/topico2/tratamientos/ HTTP/1.1" 200 361
[2016-11-27 17:18:52] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:52] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:52] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_consulta"."id", "topico2_consulta"."historia_id", "topico2_consulta"."enfermedadactual", "topico2_consulta"."examenfisico", "topico2_consulta"."fecha", "topico2_consulta"."suministrar", "topico2_consulta"."estado" FROM "topico2_consulta"; args=()
[2016-11-27 17:18:52] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_diagnostico"."id", "topico2_diagnostico"."codigo", "topico2_diagnostico"."nombre" FROM "topico2_diagnostico" INNER JOIN "topico2_consulta_diagnostico" ON ("topico2_diagnostico"."id" = "topico2_consulta_diagnostico"."diagnostico_id") WHERE "topico2_consulta_diagnostico"."consulta_id" = 4; args=(4,)
[2016-11-27 17:18:52] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_diagnostico"."id", "topico2_diagnostico"."codigo", "topico2_diagnostico"."nombre" FROM "topico2_diagnostico" INNER JOIN "topico2_consulta_diagnostico" ON ("topico2_diagnostico"."id" = "topico2_consulta_diagnostico"."diagnostico_id") WHERE "topico2_consulta_diagnostico"."consulta_id" = 5; args=(5,)
[2016-11-27 17:18:52] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_diagnostico"."id", "topico2_diagnostico"."codigo", "topico2_diagnostico"."nombre" FROM "topico2_diagnostico" INNER JOIN "topico2_consulta_diagnostico" ON ("topico2_diagnostico"."id" = "topico2_consulta_diagnostico"."diagnostico_id") WHERE "topico2_consulta_diagnostico"."consulta_id" = 6; args=(6,)
[2016-11-27 17:18:52] INFO [django.server:131] "GET /api/topico2/consultas/ HTTP/1.1" 200 540
[2016-11-27 17:18:54] DEBUG [django.db.backends:90] (0.003) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:54] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:54] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_historia"."id", "topico2_historia"."numero", "topico2_historia"."atencion", "topico2_historia"."estado", "topico2_historia"."created_at", "topico2_historia"."updated_at" FROM "topico2_historia"; args=()
[2016-11-27 17:18:54] INFO [django.server:131] "GET /api/topico2/historias/ HTTP/1.1" 200 1433
[2016-11-27 17:18:59] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:59] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:18:59] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_consulta"."id", "topico2_consulta"."historia_id", "topico2_consulta"."enfermedadactual", "topico2_consulta"."examenfisico", "topico2_consulta"."fecha", "topico2_consulta"."suministrar", "topico2_consulta"."estado" FROM "topico2_consulta"; args=()
[2016-11-27 17:18:59] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_diagnostico"."id", "topico2_diagnostico"."codigo", "topico2_diagnostico"."nombre" FROM "topico2_diagnostico" INNER JOIN "topico2_consulta_diagnostico" ON ("topico2_diagnostico"."id" = "topico2_consulta_diagnostico"."diagnostico_id") WHERE "topico2_consulta_diagnostico"."consulta_id" = 4; args=(4,)
[2016-11-27 17:18:59] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_diagnostico"."id", "topico2_diagnostico"."codigo", "topico2_diagnostico"."nombre" FROM "topico2_diagnostico" INNER JOIN "topico2_consulta_diagnostico" ON ("topico2_diagnostico"."id" = "topico2_consulta_diagnostico"."diagnostico_id") WHERE "topico2_consulta_diagnostico"."consulta_id" = 5; args=(5,)
[2016-11-27 17:18:59] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_diagnostico"."id", "topico2_diagnostico"."codigo", "topico2_diagnostico"."nombre" FROM "topico2_diagnostico" INNER JOIN "topico2_consulta_diagnostico" ON ("topico2_diagnostico"."id" = "topico2_consulta_diagnostico"."diagnostico_id") WHERE "topico2_consulta_diagnostico"."consulta_id" = 6; args=(6,)
[2016-11-27 17:18:59] INFO [django.server:131] "GET /api/topico2/consultas/ HTTP/1.1" 200 540
[2016-11-27 17:19:00] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:19:00] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:19:00] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_tratamiento"."id", "topico2_tratamiento"."consulta_id", "topico2_tratamiento"."diagnostico", "topico2_tratamiento"."periododescanso", "topico2_tratamiento"."fecha", "topico2_tratamiento"."citafecha", "topico2_tratamiento"."aministracion" FROM "topico2_tratamiento"; args=()
[2016-11-27 17:19:00] INFO [django.server:131] "GET /api/topico2/tratamientos/ HTTP/1.1" 200 361
[2016-11-27 17:19:00] INFO [django.server:131] "OPTIONS /api/topico2/especificacionrecetas/ HTTP/1.1" 200 0
[2016-11-27 17:19:00] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:19:00] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:19:00] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_especificacionreceta"."id", "topico2_especificacionreceta"."tratamiento_id", "topico2_especificacionreceta"."medicamento_id", "topico2_especificacionreceta"."cantidad", "topico2_especificacionreceta"."dosis", "topico2_especificacionreceta"."periodo", "topico2_especificacionreceta"."recomendacion", "topico2_especificacionreceta"."importe", "topico2_especificacionreceta"."precio_venta", "topico2_especificacionreceta"."created_at", "topico2_especificacionreceta"."updated_at" FROM "topico2_especificacionreceta"; args=()
[2016-11-27 17:19:00] INFO [django.server:131] "GET /api/topico2/especificacionrecetas/ HTTP/1.1" 200 499
[2016-11-27 17:19:02] INFO [django.server:131] "OPTIONS /api/topico2/medicamentos/ HTTP/1.1" 200 0
[2016-11-27 17:19:02] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:19:02] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:19:02] DEBUG [django.db.backends:90] (0.001) SELECT "topico2_medicamento"."id", "topico2_medicamento"."nombre", "topico2_medicamento"."preciocompra", "topico2_medicamento"."stock", "topico2_medicamento"."fecha", "topico2_medicamento"."tipo", "topico2_medicamento"."created_at", "topico2_medicamento"."updated_at" FROM "topico2_medicamento"; args=()
[2016-11-27 17:19:02] INFO [django.server:131] "GET /api/topico2/medicamentos/ HTTP/1.1" 200 1412
[2016-11-27 17:19:03] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:19:03] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:19:03] DEBUG [django.db.backends:90] (0.001) SELECT "topico2_diagnostico"."id", "topico2_diagnostico"."codigo", "topico2_diagnostico"."nombre" FROM "topico2_diagnostico"; args=()
[2016-11-27 17:19:03] INFO [django.server:131] "GET /api/topico2/diagnosticos/ HTTP/1.1" 200 254
[2016-11-27 17:19:04] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:19:04] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2'; args=('GAxvQzSRrF6WLHPAZZnSO6PEIcwmS2',)
[2016-11-27 17:19:04] DEBUG [django.db.backends:90] (0.000) SELECT "topico2_medicamento"."id", "topico2_medicamento"."nombre", "topico2_medicamento"."preciocompra", "topico2_medicamento"."stock", "topico2_medicamento"."fecha", "topico2_medicamento"."tipo", "topico2_medicamento"."created_at", "topico2_medicamento"."updated_at" FROM "topico2_medicamento"; args=()
[2016-11-27 17:19:04] INFO [django.server:131] "GET /api/topico2/medicamentos/ HTTP/1.1" 200 1412
